android/src/com/mapswithme/maps/help/HelpFragment.java

```diff
@@ -103,51 +104,36 @@ private void onTermOfUseClick()
   @Override
   public void onClick(View v)
   {
-    switch (v.getId())
-    {
-    case R.id.web:
-      startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(Constants.Url.WEB_SITE)));
-      break;
-
-    case R.id.github:
-      startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(Constants.Url.GITHUB)));
-      break;
-
-    case R.id.telegram:
-      startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(Constants.Url.TELEGRAM)));
-      break;
-
-    case R.id.instagram:
-      startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(Constants.Url.INSTAGRAM)));
-      break;
-
-    case R.id.facebook:
+    final int id = v.getId();
+    if (id == R.id.web)
```

Comment:
what is wrong with switch-case here?

Reply:
https://issuetracker.google.com/issues/159878591#comment3